packages/host/app/services/matrix-service.ts

```diff
@@ -1963,6 +2001,35 @@ export default class MatrixService extends Service {
     ]);
   }
 
+  private ensureRoomDraftHydrated(roomId: string) {
```

Comment:
Since we are restoring from localstorage can we simply use that nomenclature? I.e. "ensureMessageDraftRestored"?

To me hydration means more something like when you take raw html and then attach event handlers in the DOM. I think this is the traditional meaning of hydration, so it could lead to confusion when looking at these method names.